README.md

```diff
@@ -250,6 +251,11 @@ var Foo = require('foo'); // "declarationKeyword": "var"
 const Foo = require('foo'); // "declarationKeyword": "const"
 ```
 
+### `globals`
```

Comment:
The same way as other configuration options, you would have to specify the config option as a function.